scene/gui/graph_edit.cpp

```diff
@@ -42,8 +42,13 @@
 
 #define ZOOM_SCALE 1.2
 
-#define MIN_ZOOM (((1 / ZOOM_SCALE) / ZOOM_SCALE) / ZOOM_SCALE)
-#define MAX_ZOOM (1 * ZOOM_SCALE * ZOOM_SCALE * ZOOM_SCALE)
+// Allow dezooming 8 times from the default zoom level.
+// At low zoom levels, text is unreadable due to its small size and poor filtering,
+// but this is still useful for previewing purposes.
+#define MIN_ZOOM (1 / Math::pow(ZOOM_SCALE, 8))
+
+// Allow zooming 4 times from the default zoom level.
```

Comment:
Might as well add only 1 comment saying which part allows zooming how many times? That might help a bit.

Reply:
I think it's obvious enough from the number in the comment matching the `Math::pow()` exponent.